### DISCUSSION TITLE
Post your LinearMouse settings!

### DISCUSSION THREAD

**Discussion by MolinDiscovery** [linearmouse/linearmouse#788] [Show and tell]
Hi,

So I spent quite a bit of time setting up my mouse and thought we should share settings information with each other? My mouse is a Logitech MX Master 3S and I found that my settings are applicable across multiple monitors. Therefore, I believe there could be room for sharing this information.

**Logitech Master MX 3S**
```
Pointer acceleration: 2
Pointer speed 0,08

Scrolling acceleration (V): 0,6
Scrolling speed: (V): 11

Scrolling acceleration (H): 2
Scrolling speed: (H): 0,2
```

Remember, if you've already set up custom settings for individual monitors you should delete the _linearmouse.json_ file and redo your settings (preferentially just move it to a backup folder).

Hope you can use it.
Jacob

**Comment by wfghzht** [comment]
不错
